server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileService.java

```diff
@@ -183,16 +183,16 @@ public void rename(String key, String newName) {
     factory.rename(key, newName);
   }
 
-  /**
-   * Set or unset parent profile.
-   *
-   * @param key       key of existing profile
-   * @param parentKey key of parent profile to be inherited from. Or <code>null</code> to unset the parent.
-   */
-  public void setParent(String key, @Nullable String parentKey) {
-    verifyAdminPermission();
-    ruleActivator.setParent(key, parentKey);
-  }
+//  /**
+//   * Set or unset parent profile.
+//   *
+//   * @param key       key of existing profile
+//   * @param parentKey key of parent profile to be inherited from. Or <code>null</code> to unset the parent.
+//   */
+//  public void setParent(String key, @Nullable String parentKey) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)